### ISSUE TITLE
Missing Link - Recursion II (Backtracking)

### ISSUE THREAD

**Issue by lakpro** [LeetCode-Feedback/LeetCode-Feedback#12485]
#### Your LeetCode username
https://leetcode.com/Lakpro/

#### Category of the bug
- [ ] Question
- [ ] Solution
- [x] Language
- [ ] Missing Test Cases 

#### Description of the bug
In the explore card of recursion II and in the backtracking section, you should have attached a link to the wiki page in the intro section. You can take a look at the pic for reference. One has to click the link at the bottom of the page, which is not the case in other sections.Here's the link to the page: https://leetcode.com/explore/learn/card/recursion-ii/472/backtracking/2654/

#### Code you used for Submit/Run operation
NIL

#### Language used for code
NIL

#### Expected behavior
NIL

#### Screenshots
![image](https://user-images.githubusercontent.com/73340463/226449731-1b5426ef-828c-44a0-a644-cd87bc676113.png)

#### Additional context
NIL

**Comment by vivekiet22** [comment]
hi @lakpro 
I think that's the same as when you reference something in an article or anything else. That `[1]` just tells if you want to `read more` about that you can just go and `check out [1] part in the reference`.

**Comment by lakpro** [comment]
Yes, I agree, but not everyone is familiar with this notation, and it's just better and more accessible for a person to click on [1] directly. I feel like there is no harm in putting a hyperlink on that one.

**Comment by LC-Pam** [comment]
Hi there,

Thank you for reaching out to us.

 We've relayed this issue to our team to investigate. 

We will reach back to you after we have the results.

Thank you for your support.

**Comment by LC-Pam** [comment]
Thank you for your time. 

We've used your feedback to update the problem. 

Your LeetCode account has received 100 LeetCoins as a reward for this feedback. 

If you have any other questions or feedback, please don't hesitate to let us know! 

We appreciate your support!
